classes/Misc.php

```diff
@@ -2437,10 +2437,10 @@ function getServerInfo($server_id = null) {
 
 		/**
 		 * Set server information.
-		 * @param $key parameter name to set, or null to replace all
+		 * @param $key string parameter name to set, or null to replace all
```

Comment:
IDEA/PHP Storm was complaining about the paramater types not matching "expecting the but was string" etc.